webpack.config.js

```diff
@@ -14,62 +15,73 @@ const cssLoader = {
   },
 };
 
-module.exports = env => ({
-  entry: [resolve(__dirname, 'src/set-public-path.ts'), resolve(__dirname, 'src/index.ts')],
-  output: {
-    filename: 'openmrs-esm-patient-registration-app.js',
-    libraryTarget: 'system',
-    path: resolve(__dirname, 'dist'),
-    jsonpFunction: 'webpackJsonp_openmrs_esm_patient_registration',
-  },
-  module: {
-    rules: [
-      {
-        parser: {
-          system: false,
-        },
-      },
-      {
-        test: /\.m?(js|ts|tsx)$/,
-        exclude: /(node_modules|bower_components)/,
-        use: {
-          loader: 'babel-loader',
+module.exports = env => {
+  const filename = 'openmrs-esm-patient-registration-app.js';
```

Comment:
There is no need to change the layout of the whole file for this variable. The variable could be also placed module-global. No need to define it inside the function.

Reply:
Good point, I will change that.